packages/frontend/src/components/other-sites/OtherSites.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
+  return (
+    <div
+      className="
+      mt-8 flex flex-col items-center justify-between
```

Comment:
avoid multilines of classnames, if line is too long then maybe use cx module

Reply:
don't we have autoformatting for this? :O